tlatools/org.lamport.tlatools/src/tlc2/TLC.java

```diff
@@ -1564,13 +1564,17 @@ private void printUsage()
     	sharedArguments.add(new UsageGenerator.Argument("-debug",
 														"print various debugging information - not for production use\n",
 														true));
-    	sharedArguments.add(new UsageGenerator.Argument("-dump", "file",
-    													"dump all states into the specified file; this parameter takes\n"
-    														+ "optional parameters for dot graph generation. Specifying\n"
-    														+ "'dot' allows further options, comma delimited, of zero\n"
-    														+ "or more of 'actionlabels', 'colorize', 'snapshot' to be\n"
-    														+ "specified before the '.dot'-suffixed filename", true,
-    													"dot actionlabels,colorize,snapshot"));
+        sharedArguments.add(new UsageGenerator.Argument("-dump", "format file",
```

Comment:
Let's keep the the first part from the old version ("dump all states into the specified file."), followed by the new part from `current-tools.pdf`.

Reply:
I have updated my commit.